### ISSUE TITLE
Line charts not working properly in PowerPoint  in v2.1.0 of PptxGenJS

### ISSUE THREAD

**Issue by fernandacarvalho** [gitbrent/PptxGenJS#322]
Hello,
I'm trying to generate a pptx file with charts but there is an error while opening the generated file in Power Point when there is some 'chart.LINE'. Charts with '.BAR' type are rendered normally, like texts and images.

The same file opens perfectly in Keynote and in Libre Office. That's just in Power Point, and it occurs in any version (I'm using Microsoft Office 2016).
I am working with latest version of PptxGenJS (v2.1.0) and nothing goes wrong during export. I mean that the code runs normally, without erros or warnings and the file is exported. The problem occurs while trying to open.

The error messages in Power Point are:
![powerpoint-issue](https://user-images.githubusercontent.com/11180981/38703827-b6784dcc-3e7a-11e8-859d-a5dbdde3c4d0.png)

![ppt-issue](https://user-images.githubusercontent.com/11180981/38703857-ce3f2cb4-3e7a-11e8-9ca6-8d5b5904e34d.png)

![ppt-reparar](https://user-images.githubusercontent.com/11180981/38703866-d226ab0e-3e7a-11e8-988a-914431a2e451.png)

Basically a default message informing that was not possible to read some content and by trying to repair, it has been removed.

Note: By opening the file in Keynote or Google Slides and then saving/exporting as ‘pptx' it works. All charts are rendered normally.

![keynote-ex](https://user-images.githubusercontent.com/11180981/38703990-3a511264-3e7b-11e8-8be6-768bb75186b6.png)

I don’t know what’s  going on due to the lack of explicit errors and if someone could help me I would appreciate.

Thank you.

**Comment by andrei-cs** [comment]
That happened to me a lot and it was caused by missing data points in the series. 

Keynote seems to replace them with 0's correctly, but in my case, powerpoint just failed to open the presentation and showed the error above.

Hope this helps.

**Comment by gitbrent** [comment]
I agree with @andrei-cs - this looks like a data issue to me.

**Comment by szymon-jedify** [comment]
@gitbrent wtf man? it is global issue at your package! power point has more strict appproach to data model than other apps and you have a bug there, that's why it is not working!!!! several different users are submitting the same issue, and you are saying that this is a problem with data? are you mad?
